src/data/notifications.json

```diff
@@ -1,10 +1,21 @@
 {
   "notification": [
+    {
+      "title": "MayaData is celebrating Hacktoberfest 2020 with OpenEBS and LitmusChaos",
```

Comment:
Title: "OpenEBS is participating in Hacktoberfest 2020, with swag sponsored by MayaData."
URL: https://openebs.io/blog/hacktoberfest-2020-contribute-to-openebs

Reply:
thanks @kmova , it's fixed